package/skeleton-init-finit/skeleton/etc/finit.d/available/sshd.conf

```diff
@@ -1,4 +1,2 @@
-task    <pid/syslogd> \
-	[S]       /usr/bin/ssh-hostkeys                        -- Verifying SSH host keys
-service <task/ssh-hostkeys/success> env:-/etc/default/sshd \
+service <> env:-/etc/default/sshd \
```

Comment:
Please move the `<pid/syslogd>` condition to the main sshd servce now that the ssh-hostkeys start script is no longer used.